fhservices.cfg.j2

```diff
@@ -90,3 +95,13 @@ define service {
       notes This should be able to communicate with memcached on port 11211 and cannot. Check it is running and this server can communicate with it on that port.
       contact_groups rhmapadmins
 }
+
+define service {
+       service_description Storage::Pod::Disk Usage
+       hosts {{ rhmap_router_dns }}
+       check_command check_pod_disk_usage!80!90
+       use generic-service
+
+       notes One or more pods have volumes which are running out of disk.  You should clean up the files on this pod, if possible, or scale down/up the pod so that it has a fresh filesystem.
```

Comment:
Yes, it is displayed in the nagios service detail and we use it to explain where the most likely cause is (and probably later a URL for our docs with more detail)